src/ui/component/dropDownMenu/dropDownMenu.scss

```diff
@@ -0,0 +1,157 @@
+@import '@/ui/style/fonts.scss';
+@import '@/ui/style/theme.scss';
+
+.okp4-dataverse-portal-drop-down-menu-main {
+  @include with-theme {
+    .okp4-dataverse-portal-drop-down-menu-blur {
+      position: absolute;
+      top: 0;
+      height: 100%;
+      width: 100%;
+      backdrop-filter: blur(1px);
+      /* stylelint-disable-next-line property-no-vendor-prefix */
+      -webkit-backdrop-filter: blur(1px);
+    }
+
+    .okp4-dataverse-portal-drop-down-menu-container {
+      width: 100%;
+      position: relative;
+
+      .okp4-dataverse-portal-drop-down-menu-item {
+        @include noto-sans(700);
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        padding: 10px 18px;
+        color: themed('text');
+        cursor: pointer;
+        border-radius: 6px;
+        user-select: none;
+        /* stylelint-disable property-no-vendor-prefix */
+        -webkit-user-select: none;
+        -ms-user-select: none;
+        -moz-user-select: none;
+        /* stylelint-enable property-no-vendor-prefix */
```

Comment:
I don't think it is because otherwise the linter will stop enforcing the rule for the entire file. The disable should be as explicit as possible IMO

Reply:
Sorry if my feedback was not clear. This comment is displayed twice. On line 30, which is necessary to add vendor prefixes based on stylelint rule. For line 34 I don't understand why do we need it as well, since there is no prefixed properties below. Am I missing something?

Reply:
Didn't see that detail, that's why I was confused. Thanks!